.github/workflows/wheels.yml

```diff
@@ -38,16 +38,19 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-13]
+        os: [ubuntu-latest, windows-latest, macos-13, macos-14]
 
     steps:
       - uses: actions/checkout@v4
 
       - name: Install OpenMP on macOS
-        if: matrix.os == 'macos-13'
+        if: contains(matrix.os, 'macos')
         run: HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK=1 brew reinstall --build-from-source --formula ./libomp.rb
 
       - uses: pypa/cibuildwheel@v2.22.0
+        env:
+          CIBW_ARCHS_MACOS: ${{ matrix.os == 'macos-13' && 'x86_64' || matrix.os == 'macos-14' && 'arm64' || '' }}
+          CIBW_SKIP: ${{ matrix.os == 'macos-14' && 'cp38-*'|| '' }}
```

Comment:
I believe that `CIBW_SKIP` is overriding the `skip` property in `pyproject.toml`. It is causing Linux and Windows to incorrectly build for some Python versions that aren't supported.

Reply:
Sorry about that; I’ll wrangle the GitHub Actions YAML a bit and see if I can fix it.

Reply:
@ddaspit Still working on this; I was able to cover most of what was being built in https://github.com/sillsdev/thot/actions/runs/12071274111 in my last run https://github.com/jacobwegner/thot/actions/runs/12200530731; looks like I still need to dial in `CIBW_SKIP` for the `macosx-arm64` wheels.

Reply:
@ddaspit Ok, I think the latest changes are working as expected.  

https://github.com/jacobwegner/thot/actions/runs/12208693062

`macos-14` builds the following:

```
sil_thot-3.4.6-cp39-cp39-macosx_11_0_arm64.whl
sil_thot-3.4.6-cp312-cp312-macosx_11_0_arm64.whl
sil_thot-3.4.6-cp311-cp311-macosx_11_0_arm64.whl
sil_thot-3.4.6-cp310-cp310-macosx_11_0_arm64.whl
```

The previous artifacts (`cibw-wheels-ubuntu-latest-0`, `cibw-wheels-windows-latest-1` and `cibw-wheels-macos-13-2`) seem to match the artifacts from this run:

https://github.com/sillsdev/thot/actions/runs/12071274111

```
$ tree -C -L 3 --dirsfirst --sort name thot-wheels-*
thot-wheels-jacobwegner-12208693062
├── cibw-wheels-macos-13
│   ├── sil_thot-3.4.6-cp310-cp310-macosx_10_9_x86_64.whl
│   ├── sil_thot-3.4.6-cp311-cp311-macosx_10_9_x86_64.whl
│   ├── sil_thot-3.4.6-cp312-cp312-macosx_10_13_x86_64.whl
│   ├── sil_thot-3.4.6-cp38-cp38-macosx_10_9_x86_64.whl
│   └── sil_thot-3.4.6-cp39-cp39-macosx_10_9_x86_64.whl
├── cibw-wheels-macos-14
│   ├── sil_thot-3.4.6-cp310-cp310-macosx_11_0_arm64.whl
│   ├── sil_thot-3.4.6-cp311-cp311-macosx_11_0_arm64.whl
│   ├── sil_thot-3.4.6-cp312-cp312-macosx_11_0_arm64.whl
│   └── sil_thot-3.4.6-cp39-cp39-macosx_11_0_arm64.whl
├── cibw-wheels-ubuntu-latest
│   ├── sil_thot-3.4.6-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
│   ├── sil_thot-3.4.6-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
│   ├── sil_thot-3.4.6-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
│   ├── sil_thot-3.4.6-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
│   └── sil_thot-3.4.6-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
├── cibw-wheels-windows-latest
│   ├── sil_thot-3.4.6-cp310-cp310-win32.whl
│   ├── sil_thot-3.4.6-cp310-cp310-win_amd64.whl
│   ├── sil_thot-3.4.6-cp310-cp310-win_arm64.whl
│   ├── sil_thot-3.4.6-cp311-cp311-win32.whl
│   ├── sil_thot-3.4.6-cp311-cp311-win_amd64.whl
│   ├── sil_thot-3.4.6-cp311-cp311-win_arm64.whl
│   ├── sil_thot-3.4.6-cp312-cp312-win32.whl
│   ├── sil_thot-3.4.6-cp312-cp312-win_amd64.whl
│   ├── sil_thot-3.4.6-cp312-cp312-win_arm64.whl
│   ├── sil_thot-3.4.6-cp38-cp38-win32.whl
│   ├── sil_thot-3.4.6-cp38-cp38-win_amd64.whl
│   ├── sil_thot-3.4.6-cp39-cp39-win32.whl
│   ├── sil_thot-3.4.6-cp39-cp39-win_amd64.whl
│   └── sil_thot-3.4.6-cp39-cp39-win_arm64.whl
├── cibw-wheels-macos-13.zip
├── cibw-wheels-macos-14.zip
├── cibw-wheels-ubuntu-latest.zip
└── cibw-wheels-windows-latest.zip
thot-wheels-origin-12071274111
├── cibw-wheels-macos-13
│   ├── sil_thot-3.4.6-cp310-cp310-macosx_10_9_x86_64.whl
│   ├── sil_thot-3.4.6-cp311-cp311-macosx_10_9_x86_64.whl
│   ├── sil_thot-3.4.6-cp312-cp312-macosx_10_13_x86_64.whl
│   ├── sil_thot-3.4.6-cp38-cp38-macosx_10_9_x86_64.whl
│   └── sil_thot-3.4.6-cp39-cp39-macosx_10_9_x86_64.whl
├── cibw-wheels-ubuntu-latest
│   ├── sil_thot-3.4.6-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
│   ├── sil_thot-3.4.6-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
│   ├── sil_thot-3.4.6-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
│   ├── sil_thot-3.4.6-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
│   └── sil_thot-3.4.6-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
├── cibw-wheels-windows-latest
│   ├── sil_thot-3.4.6-cp310-cp310-win32.whl
│   ├── sil_thot-3.4.6-cp310-cp310-win_amd64.whl
│   ├── sil_thot-3.4.6-cp310-cp310-win_arm64.whl
│   ├── sil_thot-3.4.6-cp311-cp311-win32.whl
│   ├── sil_thot-3.4.6-cp311-cp311-win_amd64.whl
│   ├── sil_thot-3.4.6-cp311-cp311-win_arm64.whl
│   ├── sil_thot-3.4.6-cp312-cp312-win32.whl
│   ├── sil_thot-3.4.6-cp312-cp312-win_amd64.whl
│   ├── sil_thot-3.4.6-cp312-cp312-win_arm64.whl
│   ├── sil_thot-3.4.6-cp38-cp38-win32.whl
│   ├── sil_thot-3.4.6-cp38-cp38-win_amd64.whl
│   ├── sil_thot-3.4.6-cp39-cp39-win32.whl
│   ├── sil_thot-3.4.6-cp39-cp39-win_amd64.whl
│   └── sil_thot-3.4.6-cp39-cp39-win_arm64.whl
├── cibw-wheels-macos-13.zip
├── cibw-wheels-ubuntu-latest.zip
└── cibw-wheels-windows-latest.zip

9 directories, 59 files
```